src/components/header/Header.jsx

```diff
@@ -27,7 +27,7 @@ const Header = () => {
     } else if (e.key === "login") {
       navigate("/login");
     } else if (e.key === "apiDocs") {
-      window.location.href = '/docs';
```

Comment:
We'll have to at some point sync up our auto-formatters so that we don't have a bunch of little things like this leaking into our commits. I'm guessing you just use the default settings so I'll just need to set it up on my machine sometime. For now IDK if there is any way for you to just run the auto-formatter on the whole codebase at once. But if it is, before you start working on something you can just run that and commit all the changes that it makes as one commit with a title like 'Run auto formatting on codebase'

If thats no possible no worries; ill get my auto formatter setup sometime soon